.github/workflows/linux.yml

```diff
@@ -165,21 +165,21 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - run: ./scripts/install_zig.sh
-      - run: zig/zig build fuzz_ewah -- --seed 123
-      - run: zig/zig build fuzz_lsm_segmented_array -- --seed 123
-      - run: zig/zig build fuzz_lsm_tree -- --seed 123 --events-max 400
-      - run: zig/zig build fuzz_vsr_journal_format -- --seed 123
-      - run: zig/zig build fuzz_vsr_superblock -- --seed 123 --events-max 3
-      - run: zig/zig build fuzz_vsr_superblock_quorums -- --seed 123
-
-      # Build the fuzzers, but don't run them, since they're slow.
-      - run: zig/zig build build_fuzz_lsm_manifest_log
-      - run: zig/zig build build_fuzz_vsr_free_set
-      - run: zig/zig build build_fuzz_lsm_forest
+      - run: zig/zig build fuzz -- ewah                   --seed=123
+      - run: zig/zig build fuzz -- lsm_segmented_array    --seed=123
+      - run: zig/zig build fuzz -- lsm_tree               --seed=123 --events-max=400
+      - run: zig/zig build fuzz -- vsr_journal_format     --seed=123
+      - run: zig/zig build fuzz -- vsr_superblock         --seed=123 --events-max=3
+      - run: zig/zig build fuzz -- vsr_superblock_quorums --seed=123
+
+      # These are to slow to run on CI.
```

Comment:
Heh - so with `events-max` these aren't slow to run at all. Except for `vsr_free_set` which seems to ignore it.